doc/walkthrough.rst

```diff
@@ -0,0 +1,109 @@
+Walkthrough
+===========
+
+This document walkthroughs a simple scenario with Commissaire.
+
+Configuring a ContainerManager (Optional)
+-----------------------------------------
+If you will be using OpenShift, OCP, or Kubernetes then configuring a ContainerManager is the first thing to do.
+This essentially will tell commissaire how to communicate with your ContainerManager. When a cluster is associated
+to this ContainerManager new hosts will be automatically added into the the ContainerManager as nodes.
+
+Let's say you wanted to add a ContainerManager called ``ocp``,  which has a url of ``https://openshift.example.com``,
+and uses a token of ``aaa`` for authentication:
+
+.. include:: examples/commctl_create_container_manager.rst
+
```

Comment:
I realized after reading the rest of the document that a little `.. note::` here x-referencing the "Adding nodes" section below would have answered most of my questions.